spec/displaynames.html

```diff
@@ -245,28 +245,34 @@ <h1>Properties of Intl.DisplayNames Instances</h1>
   <emu-clause id="sec-intl-displaynames-abstracts">
     <h1>Abstract Operations for DisplayNames Objects</h1>
 
-    <emu-clause id="sec-canonicalcodefordisplaynames" aoid="CanonicalCodeForDisplayNames">
-      <h1>CanonicalCodeForDisplayNames ( _type_, _code_ )</h1>
-      <p>
-        The CanonicalCodeForDisplayNames abstract operation takes arguments _type_ (a String) and _code_ (a String). It verifies that the _code_ argument represents a well-formed code according to the _type_ argument and returns the case-regularized form of the _code_. The algorithm refers to <a href="https://www.unicode.org/reports/tr35/#Identifiers">UTS 35's Unicode Language and Locale Identifiers grammar</a>. The following steps are taken:
-      </p>
+    <emu-clause id="sec-canonicalcodefordisplaynames" type="abstract operation">
+      <h1>
+        CanonicalCodeForDisplayNames (
+          _type_: a String,
+          _code_: a String,
+        ): either a normal completion containing a String or a throw completion
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It verifies that _code_ represents a well-formed code according to _type_ and returns the case-regularized form of _code_.</dd>
+      </dl>
       <emu-alg>
         1. If _type_ is *"language"*, then
-          1. If _code_ does not match the `unicode_language_id` production, throw a *RangeError* exception.
-          1. If ! IsStructurallyValidLanguageTag(_code_) is *false*, throw a *RangeError* exception.
-          1. Return ! CanonicalizeUnicodeLocaleId(_code_).
+          1. If _code_ cannot be matched by the <code>unicode_language_id</code> language tag nonterminal, throw a *RangeError* exception.
+          1. If IsStructurallyValidLanguageTag(_code_) is *false*, throw a *RangeError* exception.
+          1. Return CanonicalizeUnicodeLocaleId(_code_).
         1. If _type_ is *"region"*, then
-          1. If _code_ does not match the `unicode_region_subtag` production, throw a *RangeError* exception.
+          1. If _code_ cannot be matched by the <code>unicode_region_subtag</code> language tag nonterminal, throw a *RangeError* exception.
```

Comment:
again, adding "language tag" here is super confusing. please just keep it as "<code>unicode_region_subtag</code>  nonterminal" and remove "language tag" here.